### DISCUSSION TITLE
The mempool split/scaling the UTXO solution?!?

### DISCUSSION THREAD

**Discussion by qlpqlp** [dogecoin/dogecoin#2772] [Help needed]
Hello to all :)

So, a lot of reading, and trying to find solutions for the use of the op_return and the Unspent transaction output (UTXO) and the spamming of mempool.

I came across this proposed solution for Bitcoin that in theory could resolve the issue on Dogecoin Blockchain, spliting the UTXO:

![image](https://user-images.githubusercontent.com/5968076/147244578-5aca0406-421b-4d52-9b37-aa4a9c8c2238.png)
![image](https://user-images.githubusercontent.com/5968076/147244596-0f4e6646-94da-429f-bca5-fd7c607ec4dd.png)
![image](https://user-images.githubusercontent.com/5968076/147244614-4407f0e2-2ff5-4faa-ae37-998c5b39e1ec.png)
![image](https://user-images.githubusercontent.com/5968076/147244636-754f969f-dc15-4aed-8eaa-70bfa9d5c219.png)
![image](https://user-images.githubusercontent.com/5968076/147244645-e773c05b-070a-45a9-b7a4-3433a4d3ace5.png)
![image](https://user-images.githubusercontent.com/5968076/147244651-594e886f-626b-40d9-bdcb-fa1d00017acf.png)
![image](https://user-images.githubusercontent.com/5968076/147244656-f6419dc6-8105-4981-b5a3-caf09d5a8eaa.png)
![image](https://user-images.githubusercontent.com/5968076/147244666-beba3277-eee5-49bc-a8aa-a63274a3b81a.png)
![image](https://user-images.githubusercontent.com/5968076/147244683-9b2c2004-785f-4930-93fd-1f56930d2b5c.png)
![image](https://user-images.githubusercontent.com/5968076/147244709-12d6e48c-245f-4a2a-a518-44fd594c56ea.png)
![image](https://user-images.githubusercontent.com/5968076/147244720-44140dc9-cc21-46a9-a742-f5839b745c05.png)

Original article: https://medium.com/primalbase/split-scale-scaling-bitcoin-by-partitioning-the-utxo-space-86d1547d9da7
Original docs: https://arxiv.org/pdf/1809.08473.pdf

Any toughts on this?

**Comment by ReverseControl** [comment]
Curious, what happens to the reward per block? How many coins does each sub-chain miner get? How many does each full node miner get?

**Comment by GiverofMemory** [comment]
May be a good idea, but what is the problem dogecoin is facing right now to necessitate this?
